storage/pebble/bootstrap.go

```diff
@@ -0,0 +1,126 @@
+package pebble
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/cockroachdb/pebble"
+	"github.com/onflow/flow-go/ledger/complete/wal"
+	"github.com/onflow/flow-go/module/component"
+	"github.com/onflow/flow-go/module/irrecoverable"
+	"github.com/rs/zerolog"
+)
+
+type Bootstrap struct {
+	checkpointDir      string
+	checkpointFileName string
+	log                zerolog.Logger
+	db                 *pebble.DB
+	leafNodeChan       chan *wal.LeafNode
+	rootHeight         uint64
+}
+
+func NewBootstrap(db *pebble.DB, checkpointFile string, rootHeight uint64, log zerolog.Logger) (*Bootstrap, error) {
```

Comment:
This should probably be called `RegisterBootstrap` or something indicating registers since it's at the root `pebble` package. The other option might be to move all of these things inside `registers`. Might be better?